autoload/neomake.vim

```diff
@@ -145,9 +145,14 @@ function! s:MakeJob(make_id, options) abort
     let cwd = get(maker, 'cwd', s:make_options[a:make_id].cwd)
     if len(cwd)
         let old_wd = getcwd()
+        if has('nvim')
+            let cd = haslocaldir() ? 'lcd' : haslocaldir(-1, 0) ? 'tcd' : 'cd'
```

Comment:
I'd suggest changing this if block to just

```vim
let cd = haslocaldir() ? 'lcd' : (exists(':tcd') == 2 && haslocaldir(-1, 0)) ? 'tcd' : 'cd'
```

This ensures the code works for older nvim versions and if vim ever merges the `:tcd` functionality.